.github/workflows/package-ci.yml

```diff
@@ -285,8 +285,6 @@ jobs:
           Copy-Item -Path build\uninstall_files.nsh -Destination dist
           Copy-Item -Recurse -Path build\parsec-* -Destination dist
           Copy-Item -Path build\winfsp-* -Destination dist
-          Copy-Item -Path windows-icon-handler\refresh-icon-handler\Release\refresh-icon-handler.dll -Destination dist
-          Copy-Item -Path windows-icon-handler\check-icon-handler\Release\check-icon-handler.dll -Destination dist
```

Comment:
They're removed because of the `move` we do in `windows/freeze_program` ?

Reply:
yes, it's done [here](https://github.com/Scille/parsec-cloud/pull/3764/files#diff-28d5b94a76fa468e5d7c2de1fa4b8a33928f75084c3f473e45f8d76effcbbdd9R191)

Reply:
:+1: